pkgs/tools/networking/udict/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, stdenv
```

Comment:
```suggestion
```

Reply:
stdenv is needed for darwin    
but darwin is only present because I generated the nix file with `nix-init`  
so not sure if it should stay